ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-templates.yml

```diff
@@ -44,6 +40,8 @@ default_templates:
       vtep_ip: 'ip_addressing/vtep-ip.j2'
```

Comment:
so for now in this PR only some templates are passed over but it looks like all have been implemented. In the proposed changes section it states that the PR only removes the default templates only in use in eos_designs_facts. Does it mean there is an issue when the template are also used elsewehere?

Reply:
Since the python lib can only be used from python (not jinja), we have to keep the templates that are used in jinja templates.
As we refactor, we can remove more of them.